modules/recon/contacts-contacts/abc.py

```diff
@@ -0,0 +1,88 @@
+from recon.core.module import BaseModule
+
+from recon.mixins.threads import ThreadingMixin
+
+from bs4 import BeautifulSoup
```

Comment:
Instead of beautiful soup, why not use something like lxml which is already installed in the framework? I've found it to be better than BS4 as well. I don't mind merging like this, but having a dependency could impact user population.

Reply:
I used BeautifulSoup for no other reason than because it was what I knew, and the skiptracer module that I drew inspiration from also used it. I'll look into converting.

Reply:
I am not too terribly worried about the number of folk that use this module right now, so I will save converting this to lxml for a future version. Maybe even a Hacktoberfest issue.